src/networking.c

```diff
@@ -70,6 +70,8 @@ typedef struct {
     int type;
     int not_type;
     /* Boolean flag to determine if the current client (`me`) should be filtered. 1 means "skip me", 0 means otherwise. */
+    /* Flag to determine if the current client (`me`) should be filtered. 1 means "skip me", 0 means otherwise.
+     * If set to -1, the filter is not applied. */
     int skipme;
```

Comment:
I don't see why it's a good idea to add the new value -1 to this. Does it make the logic simpler in any way? It just pushes the decision about the default value to to a different place in the code. I wasn't needed before.

Until now, it was done like this: In the CLIENT KILL implementation, we just initialize it to 1 and in CLIENT LIST we initialize it to 0, before we parse the filters. I think this is a very simple approach and I see no reason to change it, but I may be missing something.

(We could technically change it to a `bool` if we want to.)